src/core/lib/security/transport/secure_endpoint.h

```diff
@@ -23,12 +23,15 @@
 #include "src/core/lib/iomgr/endpoint.h"
 
 struct tsi_frame_protector;
+struct tsi_zero_copy_grpc_protector;
 
 extern grpc_tracer_flag grpc_trace_secure_endpoint;
 
-/* Takes ownership of protector and to_wrap, and refs leftover_slices. */
+/* Takes ownership of protector, grpc_protector, and to_wrap, and refs
```

Comment:
Let's also document that if `zero_copy_protector` is non-NULL, `protector` will never be used.

Reply:
DONE.